nsiqcppstyle_types.py

```diff
@@ -1,69 +1,45 @@
-# in the documentation and/or other materials provided with the
-# distribution.
-#
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# SPDX-License-Identifier: BSD-3-Clause
 
 from typing import NewType, Union
 import nsiqcppstyle_lexer
 import nsiqcppstyle_checker
 
 # A stack showing the lexical context the token resides in
-ContextStackType = NewType('ContextStackType', nsiqcppstyle_checker.ContextStack)
+ContextStack = NewType('ContextStack', nsiqcppstyle_checker.ContextStack)
 
 # The new context entry that will be pushed to the contextStack due to this token
-ContextType = NewType('ContextType', nsiqcppstyle_checker.Context)
+Context = NewType('Context', nsiqcppstyle_checker.Context)
 
 # A boolean flag denoting whether the function is a declaration (True) or definition (False)
-DeclarationType = NewType('DeclarationType', bool)
+Declaration = NewType('Declaration', bool)
 
 # A directory name
-DirNameType = NewType('DirNameType', str)
+DirName = NewType('DirName', str)
 
 # The name of the file being analyzed
-FileNameType = NewType('FileNameType', str)
+FileName = NewType('FileName', str)
 
 # A full function name
-FullNameType = NewType('FullNameType', str)
+FullFunctionName = NewType('FullFunctionName', str)
 
 # The lexer object used to analyze the source file
-LexerType = NewType('LexerType', nsiqcppstyle_checker.CppLexerNavigator)
+Lexer = NewType('Lexer', nsiqcppstyle_checker.CppLexerNavigator)
 
 # The line number (> 0) of the <LineType> in the file currently being processed
-LineNumberType = NewType('LineNumberType', int)
+LineNumber = NewType('LineNumber', int)
 
 # The text of the source file line just read
-LineType = NewType('LineType', str)
+LineText = NewType('LineText', str)
 
 # The target directory currently being analyzed
-TargetPathType = NewType('TargetPathType', Union[str, bytes])
+TargetDirectory = NewType('TargetDirectory', Union[str, bytes])
 
 # The token currently being processed
-TokenType = NewType('ContextStackType', nsiqcppstyle_lexer.LexToken)
-
-# The new context entry that will be pushed to the contextStack due to this "type" token
-TypeContextType = NewType('TypeContextType', ContextType)
+Token = NewType('Token', nsiqcppstyle_lexer.LexToken)
```

Comment:
same here

Reply:
See previous comment response.

Reply:
See the previous conversation, awaiting your call or to resolve this conversation.

Reply:
See previous comment.